src/core/TESTING.md

```diff
@@ -272,7 +272,7 @@ The tests cover:
 - authenticated / non-authenticated user access (when applicable)
 
 ```typescript
-// TODO after https://github.com/elastic/kibana/pull/53208/
```

Comment:
Can we just remove lines 274-276 completely, and the commit message + description might be a little too broad when looking from an overview.